common/api/backend-itwin-client.api.md

```diff
@@ -55,7 +54,7 @@ export class AzureFileHandler implements FileHandler {
     uploadFile(requestContext: AuthorizedClientRequestContext, uploadUrlString: string, uploadFromPathname: string, progressCallback?: ProgressCallback): Promise<void>;
     }
 
-// @beta
+// @public
 export abstract class BackendAuthorizationClient extends ImsAuthorizationClient {

```

Comment:
@ramanujam-raman any reason we need both the `BackendAuthorizationClient` and the `AgentAuthorizationClient`?  Both are specifically for handling the client credentials workflow with IMS that we now describe as an "Service" App type on developer.bentley.com.

It seems those two types could be merged together to simplify the API around client creds, especially when it comes to the duplicate configuration types (i.e. `AgentAuthorizationClientConfiguration` and `BackendAuthorizationClientConfiguration`.